centreon/www/include/options/accessLists/reloadACL/reloadACL.php

```diff
@@ -42,69 +42,85 @@
 require_once "./class/centreonMsg.class.php";
 
 if (isset($_GET["o"]) && $_GET["o"] === 'r') {
-    $sid = session_id();
-    $pearDB->query("UPDATE session SET update_acl = '1' WHERE session_id = '" . $pearDB->escape($sid) . "'");
-    $pearDB->query("UPDATE acl_resources SET changed = '1'");
     $msg = new CentreonMsg();
-    $msg->setTextStyle("bold");
-    $msg->setText(_("ACL reloaded"));
-    $msg->setTimeOut("3");
-    passthru(_CENTREON_PATH_ . "/cron/centAcl.php");
+    $sid = session_id();
+    try {
+        $statement = $pearDB->prepareQuery(
+            <<<SQL
+                UPDATE session SET update_acl = '1' WHERE session_id = :sessionId
+                SQL
+        );
+        $pearDB->executePreparedQuery($statement, ['sessionId' => $sid]);
```

Comment:
You should no longer use the methods of the centeronDB class, but those of the extended DBO class.

Reply:
Use the PDO methods directly.
CentreonDB method will be deprecated for new dev.